src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/button-has-type */
+/* eslint-disable no-console */
+import ReactDOM from 'react-dom';
+import Calculator from './components/calculator';
+import './index.css';
 
 function App() {
```

Comment:
- [ ] The App.js should be class-based to satisfy the project requirement below.

![Capture](https://user-images.githubusercontent.com/26796167/176660654-c0e18d39-5f80-41bd-86ca-4d0f71e131b8.PNG)

**Hint:** Your App.js should look like the code snippet below;
```javascript
import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```